actionmailer/lib/action_mailer/base.rb

```diff
@@ -392,6 +392,7 @@ module ActionMailer
   #     of an OpenSSL verify constant (<tt>'none'</tt>, <tt>'peer'</tt>, <tt>'client_once'</tt>,
   #     <tt>'fail_if_no_peer_cert'</tt>) or directly the constant (<tt>OpenSSL::SSL::VERIFY_NONE</tt>,
   #     <tt>OpenSSL::SSL::VERIFY_PEER</tt>, ...).
+  #     <tt>:ssl</tt> - Forces ActionMailer to use SSL in the SMTP connection. Defaults to <tt>false</tt>.
```

Comment:
We are also missing `:tls` for this option. Lets change to:

`<tt>:ssl/:tls</tt> Enables Action Mailer to use SMTP/TLS (SMTPS: SMTP over direct TLS connection)`

Note, there is no default.